CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+[1.9.0] - 2022-07-07
+--------------------
+
+### New features
+
+- Support the nmstate network state configuration
+
```

Comment:
For consistency with the other system roles, and to facilitate creating the collection changelog from the roles changelogs, please add sections for `Bug Fixes` and `Other Changes` like https://github.com/linux-system-roles/tlog/blob/main/CHANGELOG.md - so
```
### Bug Fixes

- IfcfgUtil: Remediate `connection_seems_active()` for controller 

### Other Changes

- use `include_tasks` instead of `include`
- make `min_ansible_version` a string in meta/main.yml
```